### PR TITLE
Fix typos in README.txt

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Base requirements:
 
 ## Linux
 
-On a **Debian** based operation execute the following steps, after cloning the repository:
+On a **Debian** based operation system execute the following steps, after cloning the repository:
```
Comment (dethos) [inline README.md]: Can you please also change `operation` to `operating`

**Review by dethos — CHANGES_REQUESTED** [review]
Well spotted. Will merge as soon as that small change is made.

File: README.md (hunk 1 above)
Comment (pfertyk) [reply README.md]: Sure, silly of me to miss this one :)

**Review by dethos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pfertyk) [reply README.md]: And of course the file's name is not README.txt but README.md, sorry for that :)

**Comment by dethos** [comment]
Everything seems to be ok. Thanks, will merge now.

**Pull request merged**